script/urlChecker.ts

```diff
@@ -22,6 +22,9 @@ const {
   useUnversionedUrls?: boolean;
 };
 
+const chunkArray = <T>(array: T[], max: number) =>
```

Comment:
There is some kind of `chunk` function. Maybe it is worth it to create new util and reuse it
https://github.com/openstax/rex-web/blob/master/src/app/content/highlights/components/SummaryPopup/ChapterFilter.tsx#L55

Reply:
Lodash also has its chunk function so i guess it's better to use that.

Reply:
yea, the chapter filter has extra logic to try to get two even columns, lodash chunk was the right thing to use here